apps/frontend/src/app/access-portal/pages/home/components/post-card/post-card.component.html

```diff
@@ -1,11 +1,16 @@
 <div class="border-neutral-6 flex flex-col rounded-sm border-4 p-2 sm:p-7">
   <div class="flex justify-between">
     <div class="flex items-center">
-      <img
-        [src]="sanitizeUrl(publication.profile.picture?.original?.url)"
-        alt="avatar example"
-        class="h-12 w-12 rounded-full object-cover"
-      />
+      <button
+        [routerLink]="['/home/profile-public']"
```

Comment:
![image](https://github.com/Venture-Miner/wagademy/assets/111710515/e5f6edd8-935b-44fd-a1e1-7d639b03a277)

Reply:
![image](https://github.com/Venture-Miner/wagademy/assets/111710515/d1059a73-3bfd-446f-9530-ca7938a54bb1)

 o problema continua

Reply:
Sorry, Gabriel! I couldn't see the distortion here, but I'll change the image to see better and test